content/en/providers/check-in/sp/_index.md

```diff
@@ -1146,6 +1194,45 @@ urn:mace:egi.eu:<GROUP-AUTHORITY>:[<GROUP>[:<SUBGROUP>:…]]:<ROLE>@<VO>
 urn:mace:egi.eu:aai.egi.eu:vm_operator@fedcloud.egi.eu
 ```
 
+### Capabilities
+
+#### Background
+
+The user's capability information is encoded in entitlements
+(`eduPersonEntitlement` attribute values in SAML or `eduperson_entitlement`
+claim in OIDC). These entitlements are typically used to indicate access rights
+to protected resources. Entitlements are multi-valued, with each value formatted
+as a URN.
```

Comment:
as a URN following the syntax defined in AARC-G027[https://aarc-community.org/guidelines/aarc-g027/]

Reply:
Done